collector/pg_stat_checkpointer.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 The Prometheus Authors
```

Comment:
I wish there was a better way to handle this.
```suggestion
// Copyright 2024 The Prometheus Authors
```

Reply:
Done!